generation_config.yaml

```diff
@@ -1,6 +1,6 @@
 gapic_generator_version: 2.37.0
 protobuf_version: '25.2'
-googleapis_commitish: 20c357f6f5c7b995b617b15cab61999e6574bdab
+googleapis_commitish: 87e7c58f3118b4c94a96dbb1f0eb53f96d2a2b54
```

Comment:
Use latest googlapis_commitish from @JoeWang1127 's PR

Reply:
I manually updated it to one of the latest commitish so I can compare `cloudcontrolspartner` with the latest from `googleapis-gen`. Done